website/docs/flink_configuration.md

```diff
@@ -3,115 +3,179 @@ title: Flink Setup
+| taskmanager.memory.task.heap.size | -- | This is the task heap memory size for TaskExecutors. This is the size of JVM heap memory reserved for write cache. | n/a | 0.9.0 |
+| taskmanager.memory.managed.size | -- | This is the managed memory size for TaskExecutors. This is the size of off-heap memory managed by the memory manager, reserved for sorting and RocksDB state backend. If you choose RocksDB as the state backend, you need to set this memory | n/a | 0.9.0 |
+
+#### Checkpoint​
+Checkpoint is a disaster recovery mechanism for Flink. When a job fails, users can choose to recover the job from the latest checkpoint to keep the latest data correctness. To keep the transaction integrity, we flush the memory buffer into a Hudi table for persistence during the checkpointing lifecycle. It’s important to note the Hudi transaction cannot commit the transaction without enabling the checkpoint config.
+
+Please read the Flink docs for [checkpoints](https://nightlies.apache.org/flink/flink-docs-master/docs/ops/state/checkpoints/) for more details. 
+
+| Property Name | Default  | Description | Scope | Since Version                          |
+|----------------|--------|----------|---------------|--------------------------------------
+| execution.checkpointing.interval | -- | Setting this value as `execution.checkpointing.interval = 150000ms`, 150000ms = 2.5min. Configuring this parameter is equivalent to enabling the checkpoint | n/a | 0.9.0|
+| state.backend | -- | This is the state backend to be used to store state. We recommend setting the store state as `rocksdb` : `state.backend: rocksdb`  | n/a | 0.9.0 |
+| state.backend.rocksdb.localdir | -- | This is the local directory (on the TaskManager) where RocksDB stores its files. | n/a | 0.9.0 |
+| state.checkpoints.dir | -- | This is the default directory used for storing the data files and meta data of checkpoints in a Flink supported filesystem. The storage path must be accessible from all participating processes/nodes(i.e., all **TaskManagers** and **JobManagers**), like the HDFS and OSS path. | n/a | 0.9.0 |
+| state.backend.incremental |  --  | This is the option whether the state backend should create incremental checkpoints. For an incremental checkpoint, only a diff from the previous checkpoint is stored, rather than the complete checkpoint state. If the store state setting is `rocksdb`, we recommend to set this config. | n/a | 0.9.0 |
+
+### Per Job Options​
+​
+You can manually configure each job with respect to your workload. For example, if your workloads change, you can manually configure more memory or parallelism. You configure each Flink job with the SQL syntax through the WITH clause. Please take a look at the example below on how to start configuring your Flink job.
```

Comment:
`You configure` -> `You can configure` ?

Reply:
is there another option in how they can configure it?